addons/point_of_sale/views/pos_combo_view.xml

```diff
@@ -1,43 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-    <record id="point_of_sale.view_pos_combo_form" model="ir.ui.view">
```

Comment:
Why don't you rename those views ids instead of deleting them?
If a customer made a child view, it will be deleted/archived instead of (hopefully) being migrated and working like before.

Reply:
Because there's a `default_available_in_pos` context in one of the views. I don't mind renaming them, but I'm not sure what will happen with this context. Maybe it's fine to ignore it?

Reply:
Yeah, the context doesn't matter, the view will be rewritten at upgrade anyway.
Renaming matters to detect custom views, either keeping or archiving them (and asking the customer to fix).
Not sure there will be custom views inheriting this one, but better safe than sorry :D.

Reply:
Done :)